addons/filesystem/functions/fnc_device_addDir.sqf

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
You can ignore the syncing comments, if this function is always executed on the server